packages/server/src/models/Model.js

```diff
@@ -578,24 +574,15 @@ export class Model extends objection.Model {
   }
 
   // @override
-  $formatJson(json) {
+  $omitFromJson() {
     const { constructor } = this
-    // Calculate and set the computed properties.
-    for (const key of constructor.computedAttributes) {
-      // Perhaps the computed property is produced in the SQL statement,
-      // in which case we don't have to do anything anymore here.
-      if (!(key in json)) {
-        const value = this[key]
-        if (value !== undefined) {
-          json[key] = value
-        }
-      }
-    }
-    // Remove hidden attributes.
-    for (const key of constructor.hiddenAttributes) {
-      delete json[key]
-    }
-    return json
+    return constructor.hiddenAttributes
```

Comment:
@ptoussai I'm not comfortable with this change. It breaks the API of `$omitFromDatabaseJson()` and Sami doesn't recommend overriding this method for such purposes: 

https://gitter.im/Vincit/objection.js?at=591ef0712b926f8a676fd2a7

Could you limit this PR to only affect computed attributes and leave the hidden attributes out, and then change the commit message to  be more specific?

This change here about the hidden attributes isn't actually a fix.